docs/ops/signals/DFT_7.md

```diff
@@ -0,0 +1,148 @@
+## DFT <a name="DFT"></a> {#openvino_docs_ops_signals_DFT_7}
+
+**Versioned name**: *DFT-7*
+
+**Category**: Signal processing
+
+**Short description**: *DFT* layer performs the discrete complex-to-complex Fourier transformation of input tensor by specified dimensions.
+
+**Attributes**:
+
+    No attributes available.
+
+**Inputs**
+
+*   **1**: `data` - Input tensor of type *T* with data for the DFT transformation. Type of elements is any supported floating point type. The last dimension of the input tensor must be equal to 2, i.e. the input tensor shape must have the form `[D_0, D_1, ..., D_{N-1}, 2]`, representing the real and imaginary components of complex numbers in `[:, ..., :, 0]` and in `[:, ..., :, 1]` correspondingly.  Required.
+*   **2**: `axes` - 1D tensor of type *T_IND* specifying dimension indices where DFT is applied, and `axes` is any unordered list of indices of different dimensions of input tensor, e.g. `[0, 4]`, `[4, 0]`, `[4, 2, 1]`, `[1, 2, 3]`. These indices should be non-negative integers from `0` to `rank(data) - 1` inclusively.  Other dimensions do not change. The order of elements in `axes` attribute matters, and mapped directly to elements in the 3d input `signal_size`. Required.
+*   **3**: `signal_size` - 1D tensor of type *T_IND* describing signal size with respect to axes from the input `axes`. For any `i in range(0, len(axes))`, if `signal_size[i] == -1`, then DFT is calculated for full size of the axis `axes[i]`. If `signal_size[i] > input_shape[axes[i]]`, then input data are zero-padded with respect to the axis `axes[i]`. Finally, `signal_size[i] < input_shape[axes[i]]`, then input data are trimmed with respect to the axis `axes[i]`. More precisely, if `signal_size[i] < input_shape[axes[i]]`, the slice `0: signal_size[i]` of the axis `axes[i]` is considered. This behaviour is [the same as in PyTorch](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/SpectralOps.cpp#L88). TensorFlow always uses full sizes of transformed axes.  Optional, with default value `[input_shape[a] for a in axes]`.
```

Comment:
Possibly simpler to use array with -1 values with the same length as axes as default.

Reply:
In such case, we aren't able to support PyTorch FFT, because PyTorch `signal_size` can contains values `-1`. See https://pytorch.org/docs/stable/fft.html#torch.fft.fftn

Reply:
Still don't understand why -1 can't be used as default values. Is signal_size[i]=-1 and signal_size[i]=input_shape[axes[i]] different things? Example with -1 in signal_size can help with this item.